azurelinuxagent/common/cgroup.py

```diff
@@ -143,7 +153,7 @@ def _get_current_cpu_total(self):
             # has not yet started (or has stopped) an associated service on a VM using systemd, the cgroup for
             # the service will not exist ('cause systemd will tear it down). This might be a transient or a
             # long-lived state, so there's no point in logging it, much less emitting telemetry.
-            pass
+            raise
```

Comment:
what is the intention of this change? the caller is still setting the value to 0

Reply:
The method `_update_cpu_data` uses this function `_get_current_cpu_total` and thus the raise is needed. The `0` is set initially, but subsequent checks would need to have this check.

Reply:
OK - you don't need the except/raise here